sdk/jdbc/azure-identity-providers-core/pom.xml

```diff
@@ -132,6 +132,10 @@
                   ${project.basedir}/javadocTemp/README.md
                 </echo>
                 <copy file="${project.basedir}/README.md" tofile="${project.basedir}/javadocTemp/README.md"/>
+                <echo>Copying ${project.basedir}/element-list to
+                  ${project.basedir}/javadocTemp/element-list
+                </echo>
+                <copy file="${project.basedir}/element-list" tofile="${project.basedir}/javadocTemp/element-list"/>
```

Comment:
Can we use a README file instead of this? The `element-list` doesn't make sense to the source code. Check this https://github.com/Azure/azure-sdk-for-java/blob/2d116c0723c859d39a8423a8bb54e38994b38173/sdk/spring/spring-cloud-azure-starter/pom.xml#L178.